docs/config.md

```diff
@@ -0,0 +1,96 @@
+# Config file options
+
+SOCI has a multitude of variables that can customize SOCI behavior to best suit users' needs. This doc will break down the possible values for each variable, where the variable can be found, and what it can do.
+
+This doc assumes the usage of a TOML file. Per TOML specs, Anything in [brackets] is the required header for the set of variables.
+
+An example configuration file can be found in config/config.toml.
+
+The following format will be used throughout this doc:
+
+## File path
+### [toml_header]
+- `toml_var_name` (type) — Brief description. Default: some value.
+
+#
+
+# Uncategorized
+
+This set of variables must be at the top of your TOML file due to not belonging to any TOML header.
+
+## config/fs.go
+### FSConfig
+- `resolve_result_entry` (int) — Max amount of entries allowed in the cache. Default: 30.
+- `debug` (bool) — Enables debugging for go-fuse in logs. This often emits sensitive data, so this should be false in production. Default: false.
+- `disable_verification` (bool) — Allows skipping TOC validation, which can give slight performance improvements if files have already been verified elsewhere. Default: false.
+- `no_prometheus` (bool) — Toggle prometheus metrics. Default: false.
+- `mount_timeout_sec` (int) — Timeout for mount if a layer can't be resolved. Default: 30.
+- `fuse_metrics_emit_wait_duration_sec` (int) — Time until snapshotter emits FUSE metrics in logs. Default: 60.
+
+## config/config.go
+### Config
+- `metrics_address` (string) — If empty, no metrics will be polled. Default: "".
+- `metrics_network` (string) — Chooses protocol to send metrics over (e.g. tcp, unix, etc). Default: "tcp".
+- `no_prometheus` — Defined [above](#configfsgofsconfig), cannot be redeclared.
```

Comment:
Oh interesting, looks like this is duplicated at the FSConfig level. This should only be at the root level. We can consider removing it from FSConfig later.